wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/homescreen/HomeFragment.kt

```diff
@@ -278,6 +278,7 @@ class HomeFragment : Fragment() {
 		binding.homescreenHeaderEmpty.root.isVisible = !hasData
 		binding.homescreenHeaderNotEmpty.root.isVisible = hasData
 		binding.homescreenListButton.isVisible = pagerItems.size > 1
+		binding.homescreenAddCertificateOptions.root.isVisible = !hasData || isAddOptionsShowing
```

Comment:
Wouldn't this show the add options every time I return to the app when I have no certs added so far?

Reply:
Yes, but the add options are always visible when the app is in an empty state. The add button is not visible in that case